internal/db/migrations-tern/001_create_table_composes.sql

```diff
@@ -3,6 +3,7 @@ CREATE TABLE IF NOT EXISTS composes(
        request jsonb NOT NULL,
        created_at timestamp NOT NULL,
        account_number varchar NOT NULL,
+       email varchar,
```

Comment:
Oh wait, this won't work actually since table already exists, this will never get executed during the migration.

You need to add a new file 005_add_email_column.sql with
```
ALTER TABLE composes ADD email varchar;
```